### ISSUE TITLE
docker run busybox ps doesn't print anything

### ISSUE THREAD

**Issue by mhagen-vmware** [vmware/vic#2894]
Running this command on latest build just hangs and doesn't return output:

```
time="2016-10-25T18:30:26Z" level=debug msg="attach connector:  Giving up on connection for 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302" 
time=2016-10-25T18:30:26.646615732Z level=debug msg=[BEGIN] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).ContainerWaitHandler:356] 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302:-1 
time="2016-10-25T18:30:26Z" level=error msg="attach connector: Connection not found error for id:42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302: context deadline exceeded" 
time=2016-10-25T18:30:26.646742249Z level=debug msg=[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Connector).Get:80] [30.002412489s] 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302 
time=2016-10-25T18:30:26.646748238Z level=debug msg=[ END ] [github.com/vmware/vic/lib/portlayer/attach.(*Server).Get:91] [30.002809809s] 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302 
time="2016-10-25T18:30:26Z" level=error msg="attach connector: Connection not found error for id:42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302: context deadline exceeded" 
time=2016-10-25T18:30:26.646760455Z level=debug msg=[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*InteractionHandlersImpl).ContainerGetStderrHandler:293] [30.003369916s] 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302 
time="2016-10-25T18:30:26Z" level=debug msg="attach connector:  Giving up on connection for 42b55147a4e93125696b07cb65733a5021ca8d07b51ef01bc8892fefce175302" 
```

[container-logs.zip](https://github.com/vmware/vic/files/551473/container-logs.zip)

**Comment by mhagen-vmware** [comment]
Running it with ps auxww doesn't hang, but doesn't return any output either.

**Comment by anchal-agrawal** [comment]
Note: an integration test in `1-06-Docker-Run` depends on this issue, please address it before closing.

**Comment by anchal-agrawal** [comment]
Possibly a dup of #1578.

**Comment by mhagen-vmware** [comment]
No longer hangs on docker run ps, also using docker run -it busybox ps partial prints the output but truncates.

**Comment by hickeng** [comment]
The initial problem should have been addressed by @caglar10ur  adding run blocking #1578 and the truncation is captured in #2660 
Closing as dup

**Comment by dougm** [comment]
Reopening, test needs to be updated before closing:
```
------------------------------------------------------------------------------
Docker run ps password check                                          | FAIL |
Test 1-6-Docker-Run.robot needs to be updated now that Issue #2894 has been resolved
------------------------------------------------------------------------------
```
